sup3r/utilities/log_interpolator.py

```diff
@@ -0,0 +1,679 @@
+"""Rescale ERA5 wind components according to log profile"""
```

Comment:
nitpicky but i would name the file `interpolate_log_profile.py` so its next to the other interp file